cmd/dist/snapshot.go

```diff
@@ -0,0 +1,325 @@
+package main
+
+import (
+	contextpkg "context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/snapshot"
+	"github.com/containerd/containerd/snapshot/overlay"
+	"github.com/urfave/cli"
+)
+
+var snapshotCommand = cli.Command{
+	Name:  "snapshot",
+	Usage: "manage snapshots",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "root directory for snapshots",
+			Value: ".snapshots",
+		},
+	},
+	Subcommands: cli.Commands{
+		snapshotListCommand,
+		snapshotPrepareCommand,
+		snapshotMountCommand,
+		snapshotViewCommand,
+		snapshotCommitCommand,
+		// snapRollbackCommand,
+		snapshotStatCommand,
+		snapshotRemoveCommand,
+	},
+}
+
+var snapshotPrepareCommand = cli.Command{
+	Name:        "prepare",
+	Usage:       "prepare a snapshot",
+	ArgsUsage:   "<key> <parent>",
+	Description: `Prepares an Active Snapshot layer [Read/Write]`,
+	Action: func(context *cli.Context) error {
+		var (
+			root   = context.GlobalString("root")
+			key    = context.Args().First()
+			parent = context.Args().Get(1)
+		)
+
+		root, err := filepath.Abs(root)
+		if err != nil {
+			return err
+		}
+		log.G(contextpkg.Background()).WithField("root", root).Debug("setting up snapshot driver")
+
+		driver, err := overlay.NewSnapshotter(root)
```

Comment:
Its done, but github diff is not showing changes here (may be, since changes are in comment)
Changes : https://github.com/containerd/containerd/pull/722/files#diff-10d209febfd0c59980bc7d16afb30a19R57